libpam/include/security/pam_sandbox.h

```diff
@@ -0,0 +1,136 @@
+#include <errno.h>
+#include <fcntl.h>
+#include <linux/landlock.h>
+#include <linux/prctl.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/prctl.h>
+#include <sys/stat.h>
+#include <sys/syscall.h>
+#include <unistd.h>
+
+#ifndef landlock_create_ruleset
+
+	static inline int
+landlock_create_ruleset(const struct landlock_ruleset_attr *const attr,
+		const size_t size, const __u32 flags)
+{
+	return syscall(__NR_landlock_create_ruleset, attr, size, flags);
```

Comment:
`__NR_landlock_create_ruleset` may not be provided by the libc headers if it is an old version.

Reply:
Those macros are detected at configuration time thanks to the configure.ac file, hence this code can't be reached if the libc headers are too old.

Reply:
Oh, I missuses the AC_CHECK_DECLS, will have to tweak it to exclude Landlock if one of the Landlock symbols is missing.